packages/react-cosmos-webpack/src/build-module-paths.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import glob from 'glob';
+
+// Modules are require calls wrapped in functions instead of plain paths
+// in order to help bundlers like Webpack know which files to bundle.
+const getRequireCb = (path) => `()=>require('${path}')`;
+
+const getFixtures = (componentsRootPath, componentCleanPath) => {
+  const fixturesRootPath = `${componentsRootPath}/__fixtures__/${componentCleanPath}`;
+
+  let fixtures = '{';
+  glob.sync(`${fixturesRootPath}/*.{js,json}`).forEach((fixturePath) => {
+    const nameMatcher = new RegExp(`^${fixturesRootPath}/(.+).(js|json)$`);
+    const matches = fixturePath.match(nameMatcher);
+    const name = matches[1];
+    const ext = matches[2];
```

Comment:
`extname`  yes, but not `basename`, which a) contains the extension and b) is read from the last `/` to the right, whereas fixtures can be namespaced. E.g. `__fixtures__/Button/simple-states/default.js`, where `matches[1]` will be `simple-states/default` and basename will be `default.js`.